_data/TravellerOnCode/TravellerOnCode.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Avijit Chakraborty
+institution: St Xaviers College Kolkata India
+profile_pic: TravellerOnCode.jpg
```

Comment:
If the file path is case-sensitive, then the file extension needs to be lowercase.

Reply:
Hi @CLiu13 ,
I just made the change. Can you please check!

Regards,
Avijit

Reply:
Thanks for the update - however, I believe that you committed the wrong changes (you added a configuration file for VS Code, instead of editing the Markdown file).

Reply:
@CLiu13 Can you check that again I have removed that file and updated the .md file. Thanks for being so patient !